decentraland/components/presentational/__snapshots__/AccountCreationProgress.test.js.snap

```diff
@@ -1,5 +1,103 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`AccountCreationProgress ProgressMessageAndLink with a message and with an action 1`] = `
+<Component
+  style={
+    Object {
+      "alignItems": "center",
+      "flexDirection": "row",
+      "justifyContent": "center",
+    }
+  }
+>
+  <Component
+    style={
+      Object {
+        "maxWidth": 600,
+      }
+    }
+  >
+    <Component
+      adjustsFontSizeToFit={true}
+      numberOfLines={2}
+    >
+      A waiting message.
+    </Component>
+  </Component>
+  <Component>
+    <LinkToBlockchain
+      action={
+        Object {
+          "getTransaction": [Function],
+        }
+      }
+    />
+  </Component>
+</Component>
+`;
+
+exports[`AccountCreationProgress ProgressMessageAndLink with a message and without an action 1`] = `
+<Component
+  style={
+    Object {
+      "alignItems": "center",
+      "flexDirection": "row",
+      "justifyContent": "center",
+    }
+  }
+>
+  <Component
+    style={
+      Object {
+        "maxWidth": 600,
+      }
+    }
+  >
+    <Component
+      adjustsFontSizeToFit={true}
+      numberOfLines={2}
+    >
+      A waiting message.
+    </Component>
+  </Component>
+  <Component>
+    <LinkToBlockchain
```

Comment:
Noting that we could have alternatively not rendered the component in the null action case as opposed to rendering it with a null action. I'm perfectly happy with the way we have it now - just mentioning it since I thought about it.